### PR TITLE
Added sushi harvest events, badger and digg balance

### PR DESCRIPTION
**Pull request by Darruma**

### CODE REVIEW

**Comment by lookfirst** [comment]
@Darruma Why bother with a PR if you're just going to merge it? There are formatting and other errors with this PR...

File: schema.graphql [hunk 1]
```diff
@@ -289,3 +289,23 @@ type FarmHarvestEvent @entity {
...
+  xSushiHarvested: BigInt!
+  totalxSushi: BigInt!
+  toStrategist: BigInt!
+  toGovernance: BigInt!
+  toBadgerTree: BigInt!
+  timestamp: BigInt!
+  blockNumber: BigInt!
+}
+
+type BadgerBalance @entity {
```
Comment (axejintao) [inline schema.graphql:304]: This is an example of how not to use a graphQL entity. 

```
type TokenBalance @entity {
  id: ID!
  balance: BigInt!
  token : Token!
}

type Token @entity {
  id: ID!
  decimals: BigInt!
  name: String!
  symbol: String!
  totalSupply: BigInt!
}
```

**Review by axejintao — COMMENTED** [review]
who should be reviewing these PR? 

there are a lot of linting errors, and general implementation things we should probably have reviewed before unilaterally opening and merging PR's.

File: schema.graphql (hunk 1 above)
Comment (axejintao) [reply schema.graphql:304]: https://thegraph.com/docs/define-a-subgraph#bad-example

Comment (axejintao) [reply schema.graphql:304]: This also applies to harvest events which can be made to do the same with generic token balances for amount harvested.

**Pull request merged**